install/envs/ubuntu.yml

```diff
@@ -8,7 +8,7 @@ channels:
 
 dependencies:
   - python=3.7
-  - numpy=1.19
+  - numpy==1.23.4
```

Comment:
Why do we need this version? There seems to be no python 3.7 compatible version of numpy 1.23, this is also the reson for the CI failure.

Reply:
I have now create a PR #1113 against this branch that takes care of the right usage of the new augmentations in the UI and in addition makes above changes to the conda files. Also removes opencv-headless, which is causing a clash with opencv in particular on the nano.

Reply:
I was getting runtime errors with numpy 1.19 which indicated I needed a newer version; I saw those issues after I added albumentations.  I'll go back to 1.19 and see if it works with albumentations.

Reply:
I removed the pinning of numpy and the install of opencv headless in my branch and that works.

Reply:
I have a bug in the way I instantiate the custom transformer.  It works great when driving from the mycar folder `python manage.py drive --model=models/linear.h5` (I have the python file in the root of the mycar folder), but it does not find it correctly when trying to train using the donkey command, 
```
Traceback (most recent call last):
  File "/Users/edmurphy/projects/temp/donkeycar/env/bin/donkey", line 11, in <module>
    load_entry_point('donkeycar', 'console_scripts', 'donkey')()
  File "/Users/edmurphy/projects/temp/donkeycar/donkeycar/management/base.py", line 626, in execute_from_command_line
    c.run(args[2:])
  File "/Users/edmurphy/projects/temp/donkeycar/donkeycar/management/base.py", line 563, in run
    args.comment)
  File "/Users/edmurphy/projects/temp/donkeycar/donkeycar/pipeline/training.py", line 139, in train
    training_pipe = BatchSequence(kl, cfg, training_records, is_train=True)
  File "/Users/edmurphy/projects/temp/donkeycar/donkeycar/pipeline/training.py", line 40, in __init__
    self.post_transformation = ImageTransformations(config, getattr(config, 'POST_TRANSFORMATIONS', []))
  File "/Users/edmurphy/projects/temp/donkeycar/donkeycar/parts/image_transformations.py", line 15, in __init__
    self.transformations = [image_transformer(name, config) for name in names]
  File "/Users/edmurphy/projects/temp/donkeycar/donkeycar/parts/image_transformations.py", line 15, in <listcomp>
    self.transformations = [image_transformer(name, config) for name in names]
  File "/Users/edmurphy/projects/temp/donkeycar/donkeycar/parts/image_transformations.py", line 98, in image_transformer
    return custom_transformer(name, config)
  File "/Users/edmurphy/projects/temp/donkeycar/donkeycar/parts/image_transformations.py", line 156, in custom_transformer
    module = __import__(module_name)
ModuleNotFoundError: No module named 'my_custom_transformer'
```
That happens even when running the training from the `mycar` folder.  I'm not sure what to do about that.

Reply:
If I create a symbolic link from the python file in the `mycar` folder into the donkeycar source folder, so from the root of the donkecar repo:
```
ln -s /Users/edmurphy/projects/temp/mycar_deep_learning/my_custom_transformer.py donkeycar/my_custom_transformer.py
```

and change the module configuration to:
```
CUSTOM_CROP_MODULE = "donkeycar.my_custom_transformer"
```
Then that will work when training using 
```
donkey train --tub=data --type=linear
```
After the training is done I'll check to see if that also works when driving.  It is a pain however.

Reply:
That does work for driving as well.